add pretty flag for human readable output

therealplato opened the pull request:
I understand this was built as a piping tool, but I find myself frequently using it for manual queries. It fails in one scenario: where I am not familiar with a large table and want to inspect a complete record (i.e. I do not know what columns I want.)

This PR adds a `-p` flag that modifies the output from a pipable format to a human-readable format that includes column names:

```
 ⎇ add-verbose-flag marianogappa/sql Ω echo "select 9 as fingers" | sql demo
9

 ⎇ add-verbose-flag marianogappa/sql Ω echo "select 9 as fingers\G" | sql demo
*************************** 1. row ***************************
9

 ⎇ add-verbose-flag marianogappa/sql Ω echo "select 9 as fingers" | sql -v demo
--------------
select 9 as fingers
--------------

+---------+
| fingers |
+---------+
|       9 |
+---------+

 ⎇ add-verbose-flag marianogappa/sql Ω echo "select 9 as fingers\G" | sql -v demo
--------------
select 9 as fingers
--------------

*************************** 1. row ***************************
fingers: 9

```

therealplato commented:
Merge conflicts resolved
bump @marianogappa

fgeller commented:
+1

marianogappa commented:
@lucianjon suggested that the verbose mode doesn't play well with the `sql all` option; I'd prefer if the verbose mode was ignored when `all` is set.

Also could we use either `-v` and `--verbose` rather than `pretty`? I think it's more common.

marianogappa commented:
Your commit has invalidated the alias `--help`; now help is only displayed with `-h`.

marianogappa commented:
I'm open to discussing changing the help text, but for this PR could you please just add the `[-v|--verbose]` conditional to the syntax?

marianogappa commented on main.go:
```diff
@@ -105,6 +110,9 @@ func runSQL(quitContext context.Context, db database, sql string, key string, pr
 	mysql := "mysql"
 	options := fmt.Sprintf(" -Nsr %v%v%v%v -e ", userOption, passOption, hostOption, db.DbName)
+	if *pretty {
+		options = fmt.Sprintf(" -vt %v%v%v%v -e ", userOption, passOption, hostOption, db.DbName)
```
If I understand this correctly, the only difference is `-Nsr` -> `-vt`. Could you please make this yet another `%v` rather than repeating the whole `options` line? i.e. creating `flagsOption` beforehand.

marianogappa commented on main.go:
```diff
@@ -23,15 +23,17 @@ type database struct {
 	Pass      string
 }
 
-var help = flag.Bool("help", false, "shows usage")
```
Note the `--help` flag doesn't work anymore.

therealplato commented:
I've fixed the --help flag.

I won't change -p to -v because -v means `version` to me and I'm already in the habit of using `-p`.

I won't change the `options := fmt.Sprintf(" -Nsr ...")` to template the mysql flags - i already found it hard to understand due to the half dozen steps in assembling the arguments to CommandContext, and I dislike obscuring yet another behavior behind a nested printf. My preference would be a single, large Sprintf that produced `mysqlArgs`, ready for `exec.CommandContext(ctx, cmdName, strings.Split(mysqlArgs))`. I gave it a shot but the quoting required for the app server case made it a nontrivial refactor.

I won't remove the help text, basically out of stubborness. I spent a long while writing clear explanations for the behavior.
